.claude/commands/REVIEW.md

```
@@ -0,0 +1,146 @@
+
+指定された GitHub プルリクエスト番号に対して、コードレビューを実行するカスタムコマンドです。
+
+## 使用方法
+
+```
+/review <PR番号>
+```
+
+例：
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)